sources/us/zip_codes_v2.hjson

```diff
@@ -1,11 +1,11 @@
 {
-  versions: '>=2'
+  versions: '2 - 6'
   production: true
   type: http
-  note: Zip Code Tabulation Areas from the Census TIGERWEB
+  note: Zip Code Tabulation Areas from the 2000 Census TIGERWEB
   wikidata: Q136208
   website: https://www.census.gov/geo/reference/zctas.html
-  data: ftp://ftp2.census.gov/geo/tiger/TIGER2017/ZCTA5/tl_2017_us_zcta510.zip
+  data: http://geocommons.com/overlays/54893
```

Comment:
this link is stale?

Reply:
Hmm, yeah. I got the link from [here](https://gist.github.com/jefffriesen/6892860). Is that a better link to use?